Sources/Networking/HTTPClient/ETagManager.swift

```diff
@@ -43,7 +44,7 @@ class ETagManager {
         withSignatureVerification: Bool,
         refreshETag: Bool = false
     ) -> [String: String] {
-        func eTag() -> String? {
+        func eTag() -> (tag: String, date: String)? {
```

Comment:
wait, how does this have a non-null date? how do we treat existing etags which don't have a date?

Reply:
The field is annotated with `@DefaultDecodable.Now`. I did that to maintain backwards compatibility, so stored etags don't fail to load when upgrading the SDK.

Reply:
so no date = now? I'm wondering if that's maybe worse than treating them as just old or something. Note that once the backend starts load shedding, it'll prefer to load shed requests that have recent cache, so no date will be more likely to be shed than something like "arbitrary old date"

Reply:
I totally missed this comment (saw it now catching up through my e-mail).

This is addressed in #2490.